catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -273,6 +326,32 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
         return staged;
     }
 
+    private List<Callable<Boolean>> parseMetacardValidators(Changeset changeset,
+            List<MetacardValidatorDefinition> metacardValidatorDefinitions) {
+        List<Callable<Boolean>> staged = new ArrayList<>();
+        BundleContext context = getBundleContext();
+        for (MetacardValidatorDefinition metacardValidatorDefinition : metacardValidatorDefinitions) {
+            try {
+                List<MetacardValidator> metacardValidators = getMetacardValidators(
+                        metacardValidatorDefinition);
+                metacardValidators.forEach(metacardValidator -> staged.add(() -> {
+                    ServiceRegistration<MetacardValidator> registration = context.registerService(
+                            MetacardValidator.class,
+                            metacardValidator,
+                            null);
+                    changeset.metacardValidatorServices.add(registration);
+                    return registration != null;
+                }));
+
+            } catch (IllegalStateException ise) {
```

Comment:
I think `context.registerService` can also throw an IllegalStateException.  Maybe make the exception more generic or have a separate try block for the service registration